### DISCUSSION TITLE
logstash导入到doris报错

### DISCUSSION THREAD

**Discussion by zixun1202** [apache/doris#30312] [A - General / Q&A]
logstash接收数据时，如果input一两条数据时加载到doris正常，filebeat推过来大量数据时logstash加载到doris报错，使用分割符分割，字段对应也是正常的

 "Message": "[INTERNAL_ERROR]too many filtered rows\n\n\t0#  std::_Function_handler<void (doris::RuntimeState*, doris::Status*), doris::StreamLoadExecutor::execute_plan_fragment(std::shared_ptr<doris::StreamLoadContext>)::$_0>::_M_invoke(std::_Any_data const&, doris::RuntimeState*&&, doris::Status*&&) at /root/src/doris-2.0/be/src/common/status.h:354\n\t1#  doris::FragmentMgr::_exec_actual(std::shared_ptr<doris::FragmentExecState>, std::function<void (doris::RuntimeState*, doris::Status*)> const&) at /var/local/ldb-toolchain/bin/../lib/gcc/x86_64-linux-gnu/11/../../../../include/c++/11/bits/unique_ptr.h:360\n\t2#  std::_Function_handler<void (), doris::FragmentMgr::exec_plan_fragment(doris::TExecPlanFragmentParams const&, std::function<void (doris::RuntimeState*, doris::Status*)> const&)::$_0>::_M_invoke(std::_Any_data const&) at /var/local/ldb-toolchain/bin/../lib/gcc/x86_64-linux-gnu/11/../../../../include/c++/11/bits/shared_ptr_base.h:701\n\t3#  doris::ThreadPool::dispatch_thread() at /root/src/doris-2.0/be/src/util/threadpool.cpp:0\n\t4#  doris::Thread::supervise_thread(void*) at /var/local/ldb-toolchain/bin/../usr/include/pthread.h:562\n\t5#  ?\n\t6#  __clone\n"

<img width="1662" alt="image" src="https://github.com/apache/doris/assets/61574030/156ea744-4df9-44b0-8331-9330760776ba">

**Comment by yagagagaga** [comment]
ErrorURL 的链接打开看看里面报错内容

**Reply by zixun1202** [reply]
<img width="1678" alt="image" src="https://github.com/apache/doris/assets/61574030/53f89a91-2235-43de-bc4a-03a0a03f41fa">
下面是我的插件输出配置
<img width="1668" alt="image" src="https://github.com/apache/doris/assets/61574030/693d1635-d3eb-4d21-96ed-f9d0f9364d93">

以及logstash的输出示例
<img width="1069" alt="image" src="https://github.com/apache/doris/assets/61574030/436d997d-e1fa-4fea-b80f-fc73ce13987b">

 数据量少的时候是正常的  filebeat推过来数据量大了就报行数不匹配 这里output的column_separator是指message字段里面的信息吗？我看之前家峰大佬的博客示例里面是把日志的message里面数据用#间隔

**Comment by Volcano139** [comment]
解决了吗？我也遇到了同样的问题
